src/org/labkey/test/LabKeySiteWrapper.java

```diff
@@ -132,7 +132,8 @@ public void simpleSignIn()
         {
             executeScript("window.onbeforeunload = null;"); // Just get logged in, ignore 'unload' alerts
             beginAt(WebTestHelper.buildURL("login", "login"));
-            waitForAnyElement("Should be on login or Home portal", Locator.id("email"), SiteNavBar.Locators.userMenu);
+            waitForAnyElement("Should be on login or Home portal", Locator.id("email"), SiteNavBar.Locators.userMenu,
+                    Locator.tagWithId("a", "user-menu-dropdown"));
```

Comment:
Add a static `getLocator` method to `org.labkey.test.components.ui.navigation.UserMenu` rather than having this "user-menu-dropdown" id defined in several places.

Reply:
Fixed, but not sure I fixed it the way you suggested. I would like to review this with you @labkey-tchad to make sure the fix is correct.